### PR TITLE
cosine distance example

### PR DESCRIPTION
**Pull request by mhmerrill**
example of using arkouda to compute cosine distance between two arkouda pdarrays.

### CODE REVIEW

File: examples/cos_dist.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+# cosine distance of two arkouda pdarrays
+# should function similarly to scipy.spatial.distance.cosine
+def ak_cos_dist(u, v):
+    return (1.0 - ak_dot(u,v)/(ak_mag2(u)*ak_mag2(v)))
+
+if __name__ == "__main__":
+    import argparse, sys, gc, time
+
+    parser = argparse.ArgumentParser(description="Example of cosine distance/similarity in arkouda")
+    parser.add_argument('hostname', help='Hostname of arkouda server')
```
Comment (hokiegeek2) [inline examples/cos_dist.py:23]: I recommend re-naming the hostname arg to server to match the Arkouda Python API

File: examples/cos_dist.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    parser.add_argument('port', type=int, help='Port of arkouda server')
+    args = parser.parse_args()
+
+    ak.v = False
+    ak.connect(server=args.hostname, port=args.port)
+
+    u1 = [1, 0, 0]
+    v1 = [0, 1, 0]
+    d1 = ak_cos_dist(ak.array(u1), ak.array(v1))
+    print("d1 = ", d1)
```
Comment (hokiegeek2) [inline examples/cos_dist.py:33]: May make sense to put assert statements in here in addition to or in lieu of print statements

File: examples/cos_dist.py (hunk 1 above)
Comment (mhmerrill) [reply examples/cos_dist.py:23]: I assume you wanted me to call this argument `'server'`

Comment (hokiegeek2) [reply examples/cos_dist.py:23]: Indeed! :)

File: examples/cos_dist.py (hunk 2 above)
Comment (mhmerrill) [reply examples/cos_dist.py:45]: added asserts and compare against `spicy.spatial.distance.cosine` result

**Pull request merged**